src/sentry/seer/code_review/tasks.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import orjson
+from django.conf import settings
+from urllib3.exceptions import HTTPError
+
+from sentry import options
+from sentry.net.http import connection_from_url
+from sentry.seer.signed_seer_api import make_signed_seer_api_request
+from sentry.silo.base import SiloMode
+from sentry.tasks.base import instrumented_task
+from sentry.taskworker.namespaces import seer_code_review_tasks
+from sentry.taskworker.retry import Retry
+from sentry.utils import metrics
+
+logger = logging.getLogger(__name__)
+
+
+# This needs to match the value defined in the Seer API:
+# https://github.com/getsentry/seer/blob/main/src/seer/routes/codegen.py
+SEER_CODEGEN_PATH = "/v1/automation/codegen"
+SEER_PR_REVIEW_RERUN_PATH = f"{SEER_CODEGEN_PATH}/rerun"
+PREFIX = "seer.code_review.check_run.rerun"
+
+
+@instrumented_task(
+    name="sentry.seer.code_review.tasks.process_github_webhook_event",
+    namespace=seer_code_review_tasks,
+    retry=Retry(times=3, delay=60),
+    silo_mode=SiloMode.REGION,
+    bind=True,  # Bind task instance to access self.request.retries for conditional logging
```

Comment:
This allows adding `self` to the signature and analyze if we have reached the maximum number of retries so we can log errors then.